src/components/PlayersComponent.tsx

```diff
@@ -1,25 +1,22 @@
 import * as React from "react";
 import { useState } from "react";
-
 import { Button, Pane, TextInput } from "evergreen-ui";
 import { Quiz, AppMethods } from "../store/types";
-
-import { useHistory } from "react-router";
+import { PlayQuizComponent } from "./PlayQuizComponent";
 
 // Modifies the players list
 export const PlayersComponent = (props: {
   quiz: Quiz;
   index: string;
-  addPlayers: AppMethods["addPlayers"];
-  editPlayers: AppMethods["editPlayers"];
+  setPlayers: AppMethods["setPlayers"];
+  onClose: () => void;
 }) => {
   const quiz = props.quiz;
   const [newPlayerName, setNewPlayerName] = useState("");
-  const [players, setPlayers] = useState<string[]>(props.quiz.players);
-  const history = useHistory();
+  const [playersList, setPlayersList] = useState<string[]>(props.quiz.players);
 
   const onClickHandler = () => {
```

Comment:
put a better name. onClickHandler for what? call it `addNewPlayer`

Reply:
or remove it completely, since is not necessary.